pages/api/sequences.py

```diff
@@ -401,9 +404,9 @@ def post(self, request, *args, **kwargs):
                 {'detail': str(err)},
                 status=status.HTTP_400_BAD_REQUEST)
 
-        return api_response.Response(
-            serializer.errors,
-            status=status.HTTP_400_BAD_REQUEST)
+        # return api_response.Response(
```

Comment:
This is odd the method does not return any result. Maybe the code at line 399 (HTTP_201_CREATED) should be placed here.